components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/src/test/java/org/wso2/carbon/identity/cors/mgt/core/helper/CORSManagementServiceTestHelper.java

```diff
@@ -18,41 +18,32 @@
 
 package org.wso2.carbon.identity.cors.mgt.core.helper;
 
-import org.wso2.carbon.identity.configuration.mgt.core.model.Attribute;
-import org.wso2.carbon.identity.configuration.mgt.core.model.ResourceAdd;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import static org.wso2.carbon.identity.cors.mgt.core.constant.TestConstants.APP_ID_1;
-import static org.wso2.carbon.identity.cors.mgt.core.internal.Constants.TENANT_ASSOCIATION;
-
 /**
  * Helper class for CORSServiceTest.
  */
 public class CORSManagementServiceTestHelper {
 
-    public static ResourceAdd getSampleTenantResourceAdd(String origin) {
-
-        List<Attribute> attributeList = new ArrayList<>();
-        Attribute attribute = new Attribute(TENANT_ASSOCIATION, "");
-        attributeList.add(attribute);
-
-        ResourceAdd resourceAdd = new ResourceAdd();
-        resourceAdd.setName(origin);
-        resourceAdd.setAttributes(attributeList);
-        return resourceAdd;
-    }
-
-    public static ResourceAdd getSampleApplicationResourceAdd(String origin) {
-
-        List<Attribute> attributeList = new ArrayList<>();
-        Attribute attribute = new Attribute(APP_ID_1, "");
-        attributeList.add(attribute);
-
-        ResourceAdd resourceAdd = new ResourceAdd();
-        resourceAdd.setName(origin);
-        resourceAdd.setAttributes(attributeList);
-        return resourceAdd;
-    }
+//    public static ResourceAdd getSampleTenantResourceAdd(String origin) {
```

Comment:
Why this part is commented? Do we need this?

Reply:
We don't. Forgot to remove it. 
I'll remove it.